spec/integration/omniauth_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "spec_helper"
+
+describe "a user login", type: :feature do
+  include Capybara::DSL
+
+  it "login button should lead to Github authentication page" do
+    visit root_path
+
+    click_link_or_button("Login")
+
+    expect(User.last.uid).to eq('123545')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.